pkg/conduit/config.go

```diff
@@ -21,13 +21,17 @@ import (
 	"github.com/conduitio/conduit/pkg/foundation/database"
 	"github.com/conduitio/conduit/pkg/foundation/log"
 	"github.com/conduitio/conduit/pkg/plugin/connector/builtin"
+	"github.com/conduitio/conduit/pkg/schema"
 	"github.com/rs/zerolog"
 )
 
 const (
 	DBTypeBadger   = "badger"
 	DBTypePostgres = "postgres"
 	DBTypeInMemory = "inmemory"
+
+	SchemaTypeApicurio = "apicurio"
```

Comment:
We shouldn't align ourselves directly with apicurio, rather call this `external` or even `confluent`, as it needs to be a schema registry compatible with the confluent schema registry API.

Reply:
@lovromazgon good point. I wasn't aware of such compatibility. I'll rename